jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -822,6 +822,25 @@ def _make_harness(name, *, shape=(4, 4), dtype=np.float32, symmetric=False,
   for dtype in [np.float32]
 )
 
+def _make_conj_harness(name, *, shape=(3, 4), dtype=np.float32, **kwargs):
+  return Harness(f"{name}_operand={jtu.format_shape_dtype_string(shape, dtype)}_kwargs={kwargs}".replace(" ", ""),
+                 lambda x: lax.conj_p.bind(x, **kwargs),
+                 [RandArg(shape, dtype)],
+                 shape=shape,
+                 dtype=dtype,
+                 **kwargs)
+
+lax_conj = tuple( # Validate dtypes
+  _make_conj_harness("dtypes", dtype=dtype)
+  for dtype in jtu.dtypes.floating + jtu.dtypes.complex
+) + tuple( # Validate kwargs
+  _make_conj_harness("kwargs", **kwargs)
+  for kwargs in [
+    { "_input_dtype": np.float32 },             # expected kwarg for ad
+    { "random_kwarg1": 1, "random_kwarg2": 2 }, # irrelevant kwargs
```

Comment:
Is this important to test? I saw your conversation with Peter about the `**kwargs` in the implementation of `conj_p`, but this does not mean that the primitive would arise with random kwargs. Have you thought of instead replacing `**kwargs` with  an optional `input_dtype`?

Reply:
Maybe it is not very important. I only made it this way for the sake of consistency with how the primitive is implemented, i.e. the implementation in lax & this one will behave the same regardless of kwargs. I thought about modifying both this impl and the one in lax, but was reluctant to change the prototype only in jax2tf. WDYT?

Reply:
I think that we should remove this harness. This would be the only harness where we test passing random kwargs to primitives.